### PR TITLE
[EDA] Config_qa flow modification to enable 2gp feature testing

### PR DESCRIPTION
**Pull request by jofsky**
Modified the 3rd and 5th steps of `config_qa` flow to only run when EDA is not an installed package in the org.

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: cumulusci.yml [hunk 1]
```diff
@@ -222,9 +221,6 @@ tasks:
                 - record_type: "Account.HH_Account"
                 - record_type: "Account.Sports_Organization"
                 - record_type: "Account.University_Department"
-                - record_type: "{managed}Attribute__c.{managed}Credential"
```
Comment (davidmreed) [inline cumulusci.yml:225]: Is this change intended to be merged?

Comment (jofsky) [reply cumulusci.yml:225]: This is a question I have for @jstvz

Comment (jstvz) [reply cumulusci.yml:225]: We should restore these.

**Review by jstvz — APPROVED** [review]

**Review by MatthewBlanski — APPROVED** [review]
